src/pages/NotificationsPage.js

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import gql from 'graphql-tag';
+import { Query } from 'react-apollo';
+
+import Page from '../components/Page';
+import Notifications from './elements/Notifications';
+import Notification from './elements/Notification';
+import NewNotification from './elements/NewNotification';
+
+const GET_NOTIFICATION = gql`
+  query Notification($path: String) {
+    app_interface_emails_v1(path: $path) {
+      path
+      labels
+      name
+      subject
+      to {
+        users {
+          path
+          name
+        }
+      }
+      body
+    }
+  }
+`;
+
+const GET_NOTIFICATIONS = gql`
+  query Notifications {
+    app_interface_emails_v1 {
+      path
+      labels
+      name
+      subject
+      to {
+        users {
+          name
+          path
+        }
+        namespaces {
+          name
+          path
+        }
+        aws_accounts {
+          name
+          path
+        }
+      }
+      body
+    }
+  }
+`;
+
+
+const NotificationsPage = ({ location }) => {
+  const path = location.hash.substring(1);
+  // console.log(path);
+  if (path === "create-new-notification") {
+    return (
+      <NewNotification />
+    )
+  }
+
+  if (path.endsWith("yml") || path.endsWith("yaml")) {
```

Comment:
What do we do if path is not "create" and if it doesn't end with "yml" or "yaml"? Can we change this check to test if path is not empty and route it to the `Query`?